java/src/jmri/jmrix/loconet/LnReporter.java

```diff
@@ -274,7 +306,6 @@ public PhysicalLocationReporter.Direction getDirection(String rep) {
                 case "enter":
                     // NOI18N
                     // LocoNet Enter message
-                    return (PhysicalLocationReporter.Direction.ENTER);
```

Comment:
I suggest that you reformat this switch statement as follows to make it easier to read.
```
            switch (m.group(2)) {
                case "enter":    // LocoNet Enter message  // NOI18N
                case "seen":     // Lissy message.  Treat them all as "entry" messages.   // NOI18N
                    return (PhysicalLocationReporter.Direction.ENTER);
                default:
                    return (PhysicalLocationReporter.Direction.EXIT);
            }
```